src/Ng/Feed2Catalog.cs

```diff
@@ -240,6 +354,49 @@ private static DateTime ForceUTC(DateTime date)
             return lastDate;
         }
 
+        static async Task<DateTime> Deletes2Catalog(SortedList<DateTime, IList<Tuple<string, string>>> packages, Storage storage, DateTime lastCreated, DateTime lastEdited, DateTime lastDeleted, bool? createdPackages, CancellationToken cancellationToken)
```

Comment:
1. private
2. When your list of parameters becomes this long, its a good sign you want an object model passed in